src/main/resources/archetype-resources/src/test/resources/logback.xml

```diff
@@ -1,34 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
During the build you don't want to be bothered by logging statements from the application. The build itself will output the useful logging. By default therefore turn off logging of the unit tests altogether.